$NetBSD: patch-ap,v 1.3 2007/03/08 15:01:51 rillig Exp $

gcc 4.1.2 says:
Sun.c:8: error: static declaration of 'op' follows non-static declaration
Sun.h:40: error: previous declaration of 'op' was here

--- Output/solaris/Sun.c.orig	2001-04-27 00:31:34.000000000 +0200
+++ Output/solaris/Sun.c	2007-03-08 16:07:29.000000000 +0100
@@ -5,7 +5,7 @@
 #include "config.h"
 #include "xmms/i18n.h"
 
-static OutputPlugin op =
+OutputPlugin op =
 {
 	NULL,
 	NULL,
